### PR TITLE
ENH: Add Elekta files

### PR DESCRIPTION
**Pull request by larsoner**
@agramfort should we incorporate our existing MNE constants in these files? It seems weird now to have separate files for the Elekta and MNE ranges.

### CODE REVIEW

File: DictionaryTypes.txt [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+    angio             2  "Enhanced Angio MRI scan" 
+    spectroscopy      3  "MRI spectroscopy scan"
+    bold              4  "Functional MRI image"
+    diffusion         5  "Diffusion tensor volume"
+    ct                6  "Computed tomography scan"
+    segment_mask      7  "Segmentation mask"
+    distance          8  "Voxel distance map"
+    transform         9  "Transformation vectors"
+#
+# ent� nama, eivat ole enaan "strukturaalista"...
```
Comment (agramfort) [inline DictionaryTypes.txt]: :)

**Comment by agramfort** [comment]
I fear it would be slow to parse these files in Python in every mne-python import.

what do you think of a unit test that checks that these files are consistent with mne-python constants.py ?

**Comment by larsoner** [comment]
Sure. It should probably live in `mne-python`, though. That way it is done continuously over there.

**Comment by larsoner** [comment]
WDYT about moving the `_MNE.txt` data into the main files now?

**Comment by larsoner** [comment]
Talking to the MEGIN folks, the MNE files should stay separate. In it goes!

**Pull request merged**